package.json

```diff
@@ -15,8 +15,10 @@
   "publishConfig": {
     "registry": "https://registry.npmjs.org/"
   },
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "bootstrap": "lerna bootstrap --no-ci",
```

Comment:
Bootstrap is no longer supported: https://lerna.js.org/docs/legacy-package-management#replacing-your-usage-of-lerna-bootstraplerna-link

So workspaces is the recommended replacement